natgeo/mortar
# [MORTAR-183] Slider CommonJS Module Definition

To test:

- ```gulp serve```
- When ```gulp serve``` open the page, click on "Components" on the top menu, you should see the slider section on it and rendering "slider" word inside a ```<h1>``` tag.
